.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "vsicons.presets.angular": false
```

Comment:
Ugh, this is my vscode settings being committed by mistake. I'll delete the directory.